src/modules/new-request-form/fields/attachments/FileListItem.tsx

```diff
@@ -86,7 +99,11 @@ export function FileListItem({
             </FileNameWrapper>
             <Tooltip content={removeFileLabel}>
               <File.Delete
-                aria-label={removeFileLabel}
+                aria-label={t(
+                  "new-request-form.attachments.remove-file-arria-label",
+                  "Remove file: {{fileName}}",
+                  { fileName }
+                )}
```

Comment:
Can we please add `aria-describedby={undefined}` here? That way, the screen reader won't read out the "Remove file" text from the tooltip after it reads out the name of the button.

```suggestion
                aria-label={t(
                  "new-request-form.attachments.remove-file-arria-label",
                  "Remove file: {{fileName}}",
                  { fileName }
                )}
                aria-describedby={undefined}
```

* Current screen reader announcement: _Remove file: [file name], Remove file, button_
* Expected screen reader announcement: _Remove file: [file name], button_

(The word "button" comes from it being an HTML button element.)

Reply:
Ok I applied suggestion